src/Query/Builder.php

```diff
@@ -932,20 +943,9 @@ protected function compileWheres(): array
             if (isset($where['operator'])) {
                 $where['operator'] = strtolower($where['operator']);
 
-                // Operator conversions
-                $convert = [
-                    'regexp' => 'regex',
-                    'elemmatch' => 'elemMatch',
-                    'geointersects' => 'geoIntersects',
-                    'geowithin' => 'geoWithin',
-                    'nearsphere' => 'nearSphere',
-                    'maxdistance' => 'maxDistance',
-                    'centersphere' => 'centerSphere',
-                    'uniquedocs' => 'uniqueDocs',
-                ];
-
-                if (array_key_exists($where['operator'], $convert)) {
-                    $where['operator'] = $convert[$where['operator']];
+                // Convert aliased operators
+                if (isset($this->conversion[$where['operator']])) {
+                    $where['operator'] = $this->conversion[$where['operator']];
```

Comment:
We had 2 lists of converted operators. I merged them into a single in the `$conversion` property.

Reply:
Just to confirm, does this intentionally use `isset()` to ensure that an operator without an alias is left as-is and yields some error down the line when another query builder method realizes it isn't supported?

I'm OK with the error being reported later, but wanted to ensure that actually happens without us having to explicitly handle it. I think the explicit handling was only for actual builder methods ("integerRaw" was it?), which I don't think applies here since we're using `where()`.

Reply:
Operators like `all` are supported and not aliases. The `isset` is here to not work on them.
Checking if the operator is supported is done before in `invalidOperator` called by Laravel's `Illuminate\Database\Query\Builder`